python/cudf/cudf/core/reshape.py

```diff
@@ -282,9 +282,19 @@ def concat(objs, axis=0, ignore_index=False, sort=None):
                 objs, axis=axis, ignore_index=ignore_index, sort=sort
             )
     elif typ is cudf.Series:
-        return cudf.Series._concat(
-            objs, axis=axis, index=None if ignore_index else True
-        )
+        objs = [obj for obj in objs if len(obj)]
+        if len(objs) == 0:
+            return cudf.Series()
+        elif len(objs) == 1:
+            if ignore_index:
+                result = cudf.Series._concat(objs, axis=axis, index=None)
+            else:
+                result = objs[0]
+            return result
+        else:
+            return cudf.Series._concat(
+                objs, axis=axis, index=None if ignore_index else True
+            )
```

Comment:
Looks like might be able to shorten this up.
```suggestion
elif len(objs) == 1 and not ignore_index:
    return objs[0]
else:
    return cudf.Series._concat(
        objs, axis=axis, index=None if ignore_index else True
    )
```

Reply:
Thanks, Chris! This looks much cleaner. Committing it :D